CODEOWNERS

```diff
@@ -412,7 +412,6 @@ Makefile* @cilium/build
 /pkg/bpf/ @cilium/loader
 /pkg/byteorder/ @cilium/sig-datapath @cilium/api
 /pkg/cgroups/ @cilium/sig-datapath
-/pkg/checker/ @cilium/ci-structure
```

Comment:
This change should probably be part of the previous commit removing `pkg/checker`.

Reply:
good catch :+1: